scripts/run-test-proxies.sh

```diff
@@ -2,4 +2,8 @@
 
 set -euo pipefail
 
-./scripts/run-es-proxy.sh "$@" & ./scripts/run-soffia-proxy.sh "$@" & ./scripts/run-xroad-proxy.sh "$@" &
+./scripts/run-es-proxy.sh "$@" &
+./scripts/run-soffia-proxy.sh "$@" &
+./scripts/run-xroad-proxy.sh "$@" &
+
+wait
```

Comment:
This won't do anything for the background processes.
The other commands are suffixed with `&`, so the shell executes the command in the background in a subshell.
If you kill the current shell with `ctrl+C`, the subshells will continue to run.